src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -1141,27 +1218,41 @@ def braid_monodromy(f, arrangement=()):
     - ``f`` -- a polynomial with two variables, over a number field
       with an embedding in the complex numbers
 
-    - ``arrangement`` -- an optional tuple of polynomials whose product
-      equals ``f``.
+    - ``arrangement`` -- tuple (default: ``[]``); an optional tuple
+      of polynomials whose product equals ``f``.
```

Comment:
```suggestion
    - ``arrangement`` -- tuple (default: ``[]``); an optional tuple
      of polynomials whose product equals ``f``
```
It is also slightly weird to say the it should be a tuple but the default is a list.